schedule/yast/qam-yast_self_update.yaml

```diff
@@ -11,9 +11,8 @@ schedule:
   - installation/licensing/accept_license
```

Comment:
please create a new file to be used only in a copy of this test suite in MUs development as discussed offline.